README.md

```diff
@@ -64,6 +64,10 @@ Python modules are installed.
 enables cross-platform "Send to Trash" functionality.
 ([source](https://github.com/hsoft/send2trash))
 
+[notify_py](https://pypi.org/project/notify_py/)
+Cross-platform desktop notification library for Python
```

Comment:
sug: instead of "Cross-platform ....," we can say, "enables delivery of desktop notifications."